docs/docs/core/entity-service.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Entity Service
+slug: /entity-service
+tags:
+  - Strapi Service
+  - Entity Service
+---
+
+# Entity Service
+The Entity Service is a wrapper around strapi.db.query
```

Comment:
I would say something similar as in our docs, 

Strapi provides an Entity Service API, built on top of the Query Engine API (Database Layer). The Entity Service is the layer that handles Strapi's complex data structures like components and dynamic zones, and uses the Query Engine API under the hood to execute database queries.

I think it is important to mention that everything core bussiness related to strapi is inside the entity service, like compos, dz and webhooks.